core/src/main/java/org/mskcc/cbio/portal/servlet/QueryBuilder.java

```diff
@@ -274,7 +275,13 @@ protected void doPost(HttpServletRequest httpServletRequest,
             xdebug.logMsg(this, "Got Database Exception:  " + e.getMessage());
             forwardToErrorPage(httpServletRequest, httpServletResponse,
                                DB_CONNECT_ERROR, xdebug);
-        } catch (ProtocolException e) {
+        } catch (DbVersionException e) {
+            String errorMessage = "Mismatch between DB version and portal version. " + e.getMessage();
```

Comment:
The version number has been updated. I also improved the messages, adding the word "schema" to make it more clear. I did not update the property name itself though.